2021-09-30-review-apps-github-labels.md

```diff
@@ -0,0 +1,99 @@
+# Handling Review Apps using labels in GitHub PRs
+
+## Problem:
+
+Heroku's Review Apps are created automatically in many projects when a PR is created/updated. While this helps in the QA process, this is has downsides in some cases:
+
+- if the PR is a work-in-progress, the review app is probably not needed yet
+- if the PR is waiting for a review of the code, the review app is probably not needed yet
+- if the PR changes something that can't really be QA'd wit ha review app, it's not needed at all (like updating the readme, setting up docker, fixing a broken test, etc)
+- Automatically-created Review Apps are deleted after 2 days so the person doing QA has to go to heroku and re-create the review app manually if it's done after that time
```

Comment:
```suggestion
- Automatically-created Review Apps are deleted after 2 days so the person doing QA has to go to heroku and re-create the review app manually if it's more than 2 days since the Review App has been created
```